domains/navi.cawolfadmin.is-not-a.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-not-a.dev",
+    "subdomain": "navi.cawolfadmin",
+
+    "owner": {
+        "email": "camilozambranovotto@hotmail.com"
+    },
+
+    "record": {
+        "A": ["129.213.49.130"]
```

Comment:
Invalid, just redirects back to search engine.

Reply:
Hi I'm using (Coolify)[https://github.com/coollabsio/coolify] behind the IP and it has a reverse proxy. If you check on a previous domain I requested [here](https://github.com/open-domains/register/blob/main/domains/cawolfadmin.is-not-a.dev.json) it has the same IP but hitting the domain on the browser opens the coolify instance I'm running instead of redirecting to Google.